src/routes.js

```diff
@@ -48,6 +48,7 @@ export const SETTINGS_PATH = '/settings';
 export const SHARE_PATH = '/share';
 export const VALIDATION_PATH = '/validation';
 export const WELCOME_PATH = '/welcome';
+export const EDIT_PROFILE = '/editprofile';
```

Comment:
When working on the edit profile functionality the route is simply /edit. I think it is better that the route is introduced when we have the functionality

Reply:
Should I maybe remove `edit profile` from menu list?

Reply:
It is ok :) I will change it later instead if I want /edit rather than editprofile